src/main/java/uk/gov/ons/ssdc/supporttool/endpoint/ActionRuleEndpoint.java

```diff
@@ -53,18 +57,65 @@ public ActionRuleEndpoint(
     this.smsTemplateRepository = smsTemplateRepository;
   }
 
+  @GetMapping
+  public List<ActionRuleDto> findActionRulesByCollex(
+      @RequestParam(value = "collectionExercise") UUID collectionExerciseId,
+      @Value("#{request.getAttribute('userEmail')}") String userEmail) {
+
+    CollectionExercise collectionExercise =
+        collectionExerciseRepository
+            .findById(collectionExerciseId)
+            .orElseThrow(
+                () ->
+                    new ResponseStatusException(
+                        HttpStatus.BAD_REQUEST, "Collection exercise not found"));
```

Comment:
These error messages would be useful to the UI but I don't think spring puts them on the response by default, we might want to set `server.error.include-message: always` in the config.

Reply:
something has gone horribly wrong with the UI if these errors are being hit